src/runtime/components/BlogFooter.vue

```diff
@@ -235,16 +217,27 @@
 
 <script setup>
 import { ref, toRefs } from "vue";
-import config from "../config";
 import { isValidEmail } from "./../utils";
 import axios from "axios";
 import bg from "../assets/images/footer/new-bg.svg";
 
 const props = defineProps({
   mixpanel: Object,
+  apiUrl: {
+    type: String,
+    required: true,
+  },
+  social_media: {
```

Comment:
Follow naming conventions. It should be in camel case and can be `socialMediaUrl`.